scanapi/settings.py

```diff
@@ -10,7 +10,7 @@ def settings():
     if not os.path.isfile(SETTINGS_FILE):
         return DEFAULT_SETTINGS
 
-    user_config = load_yaml(SETTINGS_FILE)
+    user_config = load_config_file(SETTINGS_FILE)
```

Comment:
@astenstrasser we use the `load_config_file` here too. And not only for the API spec as I said before.